<?xml version="1.0"?>
<launch>
  <arg name="ns" default="telemax"/>
  <group ns="$(arg ns)">
    <node name="iop_client_stabilizer_driver" pkg="fkie_iop_component" type="iop_component">
        <param name="capability_group" value="iop_bridge"/>
        <param name="iop_address" value="150.64.200"/>
        <param name="name_node" value="control_client"/>
        <rosparam param="services">
          [
            fkie_iop_transport: "Transport",
            fkie_iop_events: "EventsClient",
            fkie_iop_discovery: "DiscoveryClient",
            fkie_iop_accesscontrol: "AccessControlClient",
            fkie_iop_management: "ManagementClient",
            fkie_iop_client_stabilizer_driver: "StabilizerDriverClient",
          ]
        </rosparam>
        <param name="register_own_services" value="false" />
        <param name="enable_ros_interface" value="true"/>
        <rosparam param="joint_names">[actuator_0_joint, actuator_1_joint, actuator_2_joint, actuator_3_joint]</rosparam>
        <remap from="joint_states" to="joint_states"/>
        <remap from="cmd_joint_velocities" to="~cmd_joint_velocities"/>
        <remap from="flipper_velocity_controller/command" to="~flipper_velocity_controller/command"/>
    </node>
  </group>
</launch>
